eZ/Bundle/PlatformInstallerBundle/src/Installer/DbBasedInstaller.php

```diff
@@ -35,7 +35,7 @@ public function setOutput($output)
     protected function copyConfigurationFile($source, $target)
     {
         $fs = new Filesystem();
-        $fs->copy($source, $target);
+        $fs->copy($source, $target, true);
```

Comment:
I like the idea but I think we should stay with `false` as a default to be more compatible with `copy` option. But I might be wrong? :)